src/main/java/com/fuem/repositories/OrganizerDAO.java

```diff
@@ -47,6 +47,11 @@ public class OrganizerDAO extends SQLDatabase {
             + "OFFSET ? ROWS "
             + "FETCH NEXT ? ROWS ONLY";
 
+    private static final String CHECK_PASSWORD_QUERY = "SELECT * FROM Organizer WHERE email = ? AND password = ?";
```

Comment:
- Hạn chế select *, chỉ select các cột mình muốn lấy value
- Nếu chỉ cần check tồn tại record nào không thì dùng syntax dưới này
```suggestion
    private static final String CHECK_PASSWORD_QUERY = "SELECT TOP(1) FROM Organizer WHERE email = ? AND password = ?";
```